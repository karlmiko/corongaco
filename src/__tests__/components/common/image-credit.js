import React from 'react'
import renderer from 'react-test-renderer'
import ImageCredit from '~components/common/image-credit'

describe('Components : Common: Image credit', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <ImageCredit>
          <p>Sample children</p>
        </ImageCredit>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
