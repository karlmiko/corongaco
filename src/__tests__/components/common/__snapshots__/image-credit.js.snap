// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Image credit renders correctly 1`] = `
<div
  className="credit"
>
  <p>
    Sample children
  </p>
</div>
`;
